src/query_parser/v2/parser.y

```diff
@@ -709,22 +698,44 @@ text_expr(A) ::= PERCENT PERCENT PERCENT param_term(B) PERCENT PERCENT PERCENT.
 /////////////////////////////////////////////////////////////////
 
 modifier(A) ::= MODIFIER(B) . {
+  if (B.len == 0) {
+    reportSyntaxError(ctx->status, &B, "Syntax error");
+  } else {
```

Comment:
If unreachable (`mod` rule is not accepting an empty string) better to remove unreachable code.

Reply:
Code removed.